backend/server/src/main/java/ai/verta/modeldb/experiment/FutureExperimentDAO.java

```diff
@@ -471,7 +474,7 @@ private InternalFuture<Map<String, String>> getProjectIdByExperimentId(
                   "select id, project_id from experiment where id IN (<ids>) AND deleted = :deleted")) {
             List<Map<String, String>> experimentEntitiesMap =
                 query
-                    .bind("deleted", false)
+                    .bind(ModelDBConstants.DELETED, false)
```

Comment:
I don't think this is a good change. It just obfuscates what's going on.

Reply:
Reverted